lib/index.js

```diff
@@ -2,42 +2,61 @@
 
 /* eslint-disable no-console, no-process-exit */
 
+const chalk = require('chalk');
 const spawn = require('cross-spawn');
 const path = require('path');
+const tildify = require('tildify');
 const Spinner = require('cli-spinner').Spinner;
 const promptAppDefinitionSettings = require('./prompt-app-definition');
 const createAppDefinition = require('./create-app-definition');
+const readline = require('readline');
+const stream = process.stdout;
 
 const command = process.argv[2];
 const appFolder = process.argv[3];
 
-const mainCommand = 'npx create-contentful-app';
+const localCommand = 'create-contentful-app';
+const mainCommand = `npx ${localCommand}`;
 
 function onSuccess(spinner, folder) {
   spinner.stop(true);
 
-  console.log();
-  console.log(`Success! Created ./${folder}`);
-  console.log();
-  console.log(`We suggest that you begin by running:`);
-  console.log();
-  console.log(`   cd ${folder}`);
-  console.log(`   ${mainCommand} create-definition`);
-  console.log();
+  console.log(`
+${chalk.cyan('Success!')} Created a new Contentful app in ${chalk.bold(
+    tildify(path.resolve(process.cwd(), folder))
+  )}.
+
+We suggest that you begin by running:
+
+    ${chalk.cyan(`cd ${folder}`)}
```

Comment:
Same thing here with the colours. Can we make sure it doesn't conflict? I think the `dim` stuff you used somewhere might be the way to go :)

Reply:
It's using whatever `cyan` color is specified for the current theme, so the actual color will differ between themes. Our `inquirer.prompt` is also using `chalk.cyan` behind the scenes.